tests/suites/test_suite_pk.function

```diff
@@ -299,6 +299,38 @@ exit:
 }
 /* END_CASE */
 
+/* BEGIN_CASE */
+void pk_invalid_param()
+{
+    mbedtls_pk_context ctx;
+    mbedtls_md_type_t md_alg_none = MBEDTLS_MD_NONE;
```

Comment:
I would use `MBEDTLS_MD_NONE` directly in the function calls below

Reply:
used it directly now